charts/internal/shoot-system-components/charts/csi-driver-node/templates/_csidriver.tpl

```diff
@@ -0,0 +1,9 @@
+{{- define "csi-driver-node.csidriver" -}}
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
Nit: starting from v1.19 CSIDriver should be available also under `storage.k8s.io/v1`, ref https://github.com/kubernetes/kubernetes/pull/84814.